From johfel@gmx.de Sat Jul 28 08:11:09 2007
From: "Johann Felix Soden" <johfel@gmx.de>
Date: Sat, 28 Jul 2007 17:10:33 +0200
Subject: USB: Add CanonScan LiDE30 to the quirk list
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: linux-usb-devel@lists.sourceforge.net, Oliver Neukum <oneukum@suse.de>
Message-ID: <200707281710.34611.jf@alban.soden>
Content-Disposition: inline


This patch adds CanoScan N1240U/LiDE30 (Scanner) to the list of quirky USB
devices.

Signed-off-by: Johann Felix Soden <johfel@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/quirks.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -34,6 +34,8 @@ static const struct usb_device_id usb_qu
 	{ USB_DEVICE(0x04a5, 0x207e), .driver_info = USB_QUIRK_NO_AUTOSUSPEND },
 	/* Benq S2W 3300U */
 	{ USB_DEVICE(0x04a5, 0x20b0), .driver_info = USB_QUIRK_NO_AUTOSUSPEND },
+	/* Canon, Inc. CanoScan N1240U/LiDE30 */
+	{ USB_DEVICE(0x04a9, 0x220e), .driver_info = USB_QUIRK_NO_AUTOSUSPEND },
 	/* Canon, Inc. CanoScan N650U/N656U */
 	{ USB_DEVICE(0x04a9, 0x2206), .driver_info = USB_QUIRK_NO_AUTOSUSPEND },
 	/* Canon, Inc. CanoScan 1220U */
